subsys/bluetooth/host/scan.c

```diff
@@ -535,6 +535,15 @@ void bt_hci_le_adv_ext_report(struct net_buf *buf)
 		/* Convert "Legacy" property to Extended property. */
 		adv_info.adv_props = evt->evt_type ^ BT_HCI_LE_ADV_PROP_LEGACY;
 
+		if (BT_HCI_LE_ADV_EVT_TYPE_DATA_STATUS(evt->evt_type) ==
+		    BT_HCI_LE_ADV_EVT_TYPE_DATA_STATUS_PARTIAL) {
+			/* Handling of incomplete reports is currently not
+			 * handled in the host. The remaining advertising
+			 * reports may therefore contain partial data.
+			 */
+			BT_WARN("Incomplete adv report");
+		}
```

Comment:
Given that you actually write "incomplete" and nor "partial" I am considering if this should also cover the case where the evt_type is `BT_HCI_LE_ADV_EVT_TYPE_DATA_STATUS_INCOMPLETE`?

Reply:
`BT_HCI_LE_ADV_EVT_TYPE_DATA_STATUS_PARTIAL` = "Incomplete, more data to come"
`BT_HCI_LE_ADV_EVT_TYPE_DATA_STATUS_INCOMPLETE` = "Incomplete, data truncated, no more to come"

If we only receive reports with data status set to `BT_HCI_LE_ADV_EVT_TYPE_DATA_STATUS_INCOMPLETE`, that is fine, as no reassembly is needed.